src/lua/LuaFileSystem.cpp

```diff
@@ -151,6 +191,42 @@ static int l_filesystem_join_path(lua_State *l)
 	}
 }
 
+
+/*
+ * Function: MakeUserDataDirectory
+ *
+ * > local path = FileSystem.MakeUserDataDirectory( dir_name )
+ *
+ * Creating the given directory if it's missing, returning a boolean
+ * indicating success
+ *
+ * Availability:
+ *
+ *   ????
+ *
+ * Status:
+ *
+ *   experimental
+ */
```

Comment:
regarding `????` 

I've often wondered if the "availability" and "status" fields are something we really need to specify? Might have been more relevant when we did nightly builds, and alpha-release schedule?

Reply:
At this point I'd use neither; at best the `availability` field helps modders who are writing code against the 'bleeding edge' of Pioneer, but we provide very little in terms of a stable API and there's absolutely no intention for old versions of Pioneer to remain "in use" once a new release is cut. The `status` field is actually more useful to document parts of the API that exist only for backwards compatibility or as hacks, but we should strive to remove those where present.

I do get a kick out of seeing `Status: experimental` on widely-used functions implemented in ~2013 though, it neatly proves the saying "nothing more permanent than a temporary solution".

Reply:
lol, yeah, I just put that in as I don't have any clue what to put in there and meant to ask